src/raft.c

```diff
@@ -346,8 +366,13 @@ void RaftExecuteCommandArray(RedisRaftCtx *rr,
         }
 
         enterRedisModuleCall();
-        reply = RedisModule_Call(ctx, cmd,
-                                 rr->resp_call_fmt, &c->argv[1], c->argc - 1);
+        if (user) {
+            reply = RedisModule_CallWithUser(ctx, user, cmd,
```

Comment:
Shoul we allow null user in this function? It'd effectively be `RedisModule_Call()` when user is null.

Reply:
hmm.  reasonable, makes code simpler.  I'd agree.